sql/moz-fx-data-shared-prod/firefox_accounts/fxa_all_events/view.sql

```diff
@@ -19,7 +19,7 @@ WITH fxa_auth_events AS (
   -- but should always be included for a complete raw event log.
 fxa_auth_bounce_events AS (
   SELECT
-    timestamp AS submission_timestamp,
+    `timestamp` AS submission_timestamp,
```

Comment:
Do we need backticks around every timestamp?

Reply:
I believe this is a good practice since `timestamp` would also be a keyword so this way we make sure it is not treated as such.